build.sbt

```diff
@@ -1779,14 +1795,6 @@ lazy val `bench-libs` = (project in file("std-bits/benchmarks"))
     (Compile / mainClass) := Some("org.enso.benchmarks.libs.LibBenchRunner"),
     (Compile / run / fork) := true,
     (Compile / run / connectInput) := true,
-    Compile / javacOptions := ((Compile / javacOptions).value ++
-      // Only run ServiceProvider processor and ignore those defined in META-INF, thus
-      // fixing incremental compilation setup
-      Seq(
-        "-processor",
-        "org.netbeans.modules.openide.util.ServiceProviderProcessor"
```

Comment:
Áááá... so this was the reason the processor couldn't be executed! Removing `-processor` option is good. Now the processor is going to be discovered via `META-INF/services`.

I think you can now merge the two projects into one - if you move `LibBenchRunner` into `src/bench/java`...

Reply:
This comment is outdated. We now have two projects - `std-benchmarks` and `bench-processor`.